[2025-07-01T12:29:58.936+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: northwind.carregamento_postgres manual__2025-07-01T12:23:51.035112+00:00 [queued]>
[2025-07-01T12:29:58.947+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: northwind.carregamento_postgres manual__2025-07-01T12:23:51.035112+00:00 [queued]>
[2025-07-01T12:29:58.948+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-07-01T12:29:58.962+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): carregamento_postgres> on 2025-07-01 12:23:51.035112+00:00
[2025-07-01T12:29:58.968+0000] {standard_task_runner.py:60} INFO - Started process 243 to run task
[2025-07-01T12:29:58.972+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'northwind', 'carregamento_postgres', 'manual__2025-07-01T12:23:51.035112+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmptqi8_6_k']
[2025-07-01T12:29:58.974+0000] {standard_task_runner.py:88} INFO - Job 26: Subtask carregamento_postgres
[2025-07-01T12:29:59.026+0000] {task_command.py:423} INFO - Running <TaskInstance: northwind.carregamento_postgres manual__2025-07-01T12:23:51.035112+00:00 [running]> on host 9dc97b55674e
[2025-07-01T12:29:59.173+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lrbf@cin.ufpe.br' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='northwind' AIRFLOW_CTX_TASK_ID='carregamento_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-07-01T12:23:51.035112+00:00' AIRFLOW_CTX_TRY_NUM***ER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-01T12:23:51.035112+00:00'
[2025-07-01T12:29:59.326+0000] {docker.py:359} INFO - Starting docker container from image meltano:latest
[2025-07-01T12:29:59.546+0000] {docker.py:429} INFO - Running meltano install...
[2025-07-01T12:30:01.926+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:01.925404Z[0m [[32m[1minfo     [0m] [1mInstalling 7 plugins          [0m
[2025-07-01T12:30:01.936+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:01.935482Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-postgres'[0m
[2025-07-01T12:30:01.958+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:01.957193Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-csv'[0m
[2025-07-01T12:30:01.976+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:01.975990Z[0m [[32m[1minfo     [0m] [1mInstalling extractor 'tap-csv-latest'[0m
[2025-07-01T12:30:02.004+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:02.004145Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-postgres'[0m
[2025-07-01T12:30:02.063+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:02.062441Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv'[0m
[2025-07-01T12:30:02.105+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:02.102438Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv-postgres'[0m
[2025-07-01T12:30:02.205+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:02.204439Z[0m [[32m[1minfo     [0m] [1mInstalling loader 'target-csv-csv'[0m
[2025-07-01T12:30:11.775+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:11.773896Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-postgres'[0m
[2025-07-01T12:30:11.780+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:11.780193Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-postgres'[0m
[2025-07-01T12:30:11.067+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:11.063285Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-csv-latest'[0m
[2025-07-01T12:30:12.408+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:12.407332Z[0m [[32m[1minfo     [0m] [1mInstalled extractor 'tap-csv' [0m
[2025-07-01T12:30:12.634+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:12.633455Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv' [0m
[2025-07-01T12:30:12.807+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:12.806532Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv-postgres'[0m
[2025-07-01T12:30:12.855+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:12.854812Z[0m [[32m[1minfo     [0m] [1mInstalled loader 'target-csv-csv'[0m
[2025-07-01T12:30:12.855+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:12.855140Z[0m [[32m[1minfo     [0m] [1mInstalled 7/7 plugins         [0m
[2025-07-01T12:30:14.039+0000] {docker.py:429} INFO - Meltano install completed. Executing command: meltano el tap-csv-latest target-postgres
[2025-07-01T12:30:15.167+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:15.166658Z[0m [[32m[1minfo     [0m] [1mEnvironment 'dev' is active   [0m
[2025-07-01T12:30:15.573+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:15.572741Z[0m [[32m[1minfo     [0m] [1mRunning extract & load...     [0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m
[2025-07-01T12:30:16.518+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.517541Z[0m [[32m[1minfo     [0m] [1mTraceback (most recent call last):[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.518+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.518059Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/bin/tap-csv", line 8, in <module>[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.519+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.518399Z[0m [[32m[1minfo     [0m] [1m    sys.exit(TapCSV.cli())    [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.519+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.518620Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/click/core.py", line 1161, in __call__[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.519+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.519009Z[0m [[32m[1minfo     [0m] [1m    return self.main(*args, **kwargs)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.520+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.519341Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/click/core.py", line 1082, in main[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.520+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.519590Z[0m [[32m[1minfo     [0m] [1m    rv = self.invoke(ctx)     [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.520+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.519856Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/plugin_base.py", line 93, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.521+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.520148Z[0m [[32m[1minfo     [0m] [1m    return super().invoke(ctx)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.521+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.520418Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/click/core.py", line 1443, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.521+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.520804Z[0m [[32m[1minfo     [0m] [1m    return ctx.invoke(self.callback, **ctx.params)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.521+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.521102Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/click/core.py", line 788, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.522+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.521445Z[0m [[32m[1minfo     [0m] [1m    return __callback(*args, **kwargs)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.522+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.521756Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 516, in invoke[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.523+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.522087Z[0m [[32m[1minfo     [0m] [1m    tap = cls(                [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.523+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.522427Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 118, in __init__[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.523+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.522762Z[0m [[32m[1minfo     [0m] [1m    self.setup_mapper()       [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.524+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.523214Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 194, in setup_mapper[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.524+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.523611Z[0m [[32m[1minfo     [0m] [1m    self.mapper.register_raw_streams_from_catalog(self.catalog)[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.525+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.523939Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 187, in catalog[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.525+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.524368Z[0m [[32m[1minfo     [0m] [1m    self._catalog = self.input_catalog or self._singer_catalog[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.525+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.524664Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 340, in _singer_catalog[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.525+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.525055Z[0m [[32m[1minfo     [0m] [1m    for stream in self.streams.values()[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.526+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.525322Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 149, in streams[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.526+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.525700Z[0m [[32m[1minfo     [0m] [1m    for stream in self.load_streams():[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.526+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.525994Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/singer_sdk/tap_base.py", line 373, in load_streams[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.527+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.526358Z[0m [[32m[1minfo     [0m] [1m    for stream in self.discover_streams():[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.527+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.526622Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/tap_csv/tap.py", line 89, in discover_streams[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.528+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.527041Z[0m [[32m[1minfo     [0m] [1m    return [                  [0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.528+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.527612Z[0m [[32m[1minfo     [0m] [1m  File "/opt/meltano/.meltano/extractors/tap-csv-latest/venv/lib/python3.9/site-packages/tap_csv/tap.py", line 92, in <listcomp>[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.528+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.527955Z[0m [[32m[1minfo     [0m] [1m    name=file_config.get("entity"),[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.529+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.528377Z[0m [[32m[1minfo     [0m] [1mAttributeError: 'str' object has no attribute 'get'[0m [36mcmd_type[0m=[35mextractor[0m [36mname[0m=[35mtap-csv-latest[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.685+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.684623Z[0m [[32m[1minfo     [0m] [1mtarget-postgres v0.4.3, Meltano SDK v0.47.4[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.686+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.686171Z[0m [[32m[1minfo     [0m] [1m/opt/meltano/.meltano/loaders/target-postgres/venv/lib/python3.9/site-packages/singer_sdk/plugin_base.py:715: SingerSDKDeprecationWarning: Passing a list of config file paths is deprecated. Please pass the config as a dictionary instead.[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.687+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.686445Z[0m [[32m[1minfo     [0m] [1m  super().__init__(           [0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.687+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.686642Z[0m [[32m[1minfo     [0m] [1m                              [0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.687+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.686869Z[0m [[32m[1minfo     [0m] [1mSkipping parse of env var settings...[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.688+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.687213Z[0m [[32m[1minfo     [0m] [1mThe `dialect+driver` configuration option is deprecated. Please set it to `postgresql+psycopg`, as this will be the hard-coded value in the future.[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.688+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.687426Z[0m [[32m[1minfo     [0m] [1mReader 'target-postgres' completed processing 0 lines of input (0 schemas, 0 records, 0 batch manifests, 0 state messages, 0 activate version messages).[0m [36mcmd_type[0m=[35mloader[0m [36mname[0m=[35mtarget-postgres[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.787+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.786332Z[0m [[31m[1merror    [0m] [1mExtraction failed             [0m [36mcode[0m=[35m1[0m [36mmessage[0m=[35m"AttributeError: 'str' object has no attribute 'get'"[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m
[2025-07-01T12:30:16.787+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.787184Z[0m [[32m[1minfo     [0m] [1mELT could not be completed: Extractor failed.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.787+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.787496Z[0m [[32m[1minfo     [0m] [1mFor more detailed log messages re-run the command using 'meltano --log-level=debug ...' CLI flag.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.788+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.787771Z[0m [[32m[1minfo     [0m] [1mNote that you can also check the generated log file at '/opt/meltano/.meltano/logs/elt/2025-07-01T123015--tap-csv-latest--target-postgres/98c7c547-45b7-433d-9d06-3104c63bd1c2/elt.log'.[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.788+0000] {docker.py:429} INFO - [2m2025-07-01T12:30:16.788018Z[0m [[32m[1minfo     [0m] [1mFor more information on debugging and logging: https://docs.meltano.com/reference/command-line-interface#debugging[0m [36mcmd_type[0m=[35melt[0m [36mname[0m=[35mmeltano[0m [36mrun_id[0m=[35m98c7c547-45b7-433d-9d06-3104c63bd1c2[0m [36mstate_id[0m=[35m2025-07-01T123015--tap-csv-latest--target-postgres[0m [36mstdio[0m=[35mstderr[0m
[2025-07-01T12:30:16.802+0000] {docker.py:429} INFO - Need help fixing this problem? Visit http://melta.no/ for troubleshooting steps, or to
join our friendly Slack community.

ELT could not be completed: Extractor failed.
For more detailed log messages re-run the command using 'meltano --log-level=debug ...' CLI flag.
Note that you can also check the generated log file at '/opt/meltano/.meltano/logs/elt/2025-07-01T123015--tap-csv-latest--target-postgres/98c7c547-45b7-433d-9d06-3104c63bd1c2/elt.log'.
For more information on debugging and logging: https://docs.meltano.com/reference/command-line-interface#debugging
[2025-07-01T12:30:17.877+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 376, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2025-07-01T12:30:17.880+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=northwind, task_id=carregamento_postgres, execution_date=20250701T122351, start_date=20250701T122958, end_date=20250701T123017
[2025-07-01T12:30:17.943+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/email.py:154 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2025-07-01T12:30:17.945+0000] {configuration.py:1046} WARNING - section/key [smtp/smtp_user] not found in config
[2025-07-01T12:30:17.945+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-07-01T12:30:17.962+0000] {configuration.py:1046} WARNING - section/key [smtp/smtp_user] not found in config
[2025-07-01T12:30:17.963+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-07-01T12:30:17.963+0000] {taskinstance.py:826} ERROR - Failed to send email to: ['lrbf@cin.ufpe.br']
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2334, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2499, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2516, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 376, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1000, in _email_alert
    send_email(task.email, subject, html_content)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 80, in send_email
    return backend(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 154, in send_email_smtp
    send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 272, in send_mime_email
    smtp_conn = _get_smtp_connection(smtp_host, smtp_port, smtp_timeout, smtp_ssl)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 316, in _get_smtp_connection
    return smtplib.SMTP(host=host, port=port, timeout=timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 255, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.10/smtplib.py", line 341, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 312, in _get_socket
    return socket.create_connection((host, port), timeout,
  File "/usr/local/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/local/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 824, in _handle_failure
    task_instance.email_alert(error, failure_context["task"])
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2946, in email_alert
    _email_alert(task_instance=self, exception=exception, task=task)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 1002, in _email_alert
    send_email(task.email, subject, html_content_err)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 80, in send_email
    return backend(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 154, in send_email_smtp
    send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 272, in send_mime_email
    smtp_conn = _get_smtp_connection(smtp_host, smtp_port, smtp_timeout, smtp_ssl)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/email.py", line 316, in _get_smtp_connection
    return smtplib.SMTP(host=host, port=port, timeout=timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 255, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.10/smtplib.py", line 341, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.10/smtplib.py", line 312, in _get_socket
    return socket.create_connection((host, port), timeout,
  File "/usr/local/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/local/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
[2025-07-01T12:30:17.982+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 26 for task carregamento_postgres (Docker container failed: {'StatusCode': 1}; 243)
[2025-07-01T12:30:18.002+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-07-01T12:30:18.017+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
